analysis/Analysis/Section_5_5.lean

```diff
@@ -99,63 +99,68 @@ theorem Real.upperBound_discrete_unique {E: Set Real} {n:ℕ} {m m':ℤ}
 theorem Real.LIM_of_Cauchy {q:ℕ → ℚ} (hq: ∀ M, ∀ n ≥ M, ∀ n' ≥ M, |q n - q n'| ≤ 1 / (M+1)) :
     (q:Sequence).isCauchy ∧ ∀ M, |q M - LIM q| ≤ 1 / (M+1) := by sorry
 
-/-- Theorem 5.5.9 (Existence of least upper bound)-/
```

Comment:
`LUB_exist` was timing out after the coercion changes, need to pull the long proofs out